programs/filebench/meta.yaml

```diff
@@ -0,0 +1,18 @@
+short_description: >
```

Comment:
use "Filebench is a file system and storage benchmark that can generate a large
variety of workloads."

Reply:
ok